editor/editor_inspector.cpp

```diff
@@ -1969,7 +1969,7 @@ void EditorInspector::refresh() {
 	if (refresh_countdown > 0 || changing) {
 		return;
 	}
-	refresh_countdown = EditorSettings::get_singleton()->get("docks/property_editor/auto_refresh_interval");
+	refresh_countdown = refresh_interval_cache;
```

Comment:
That means that a restart is needed to take changes into account. Is the setting defined with `EDITOR_DEF_RST`?

Reply:
No, it's just set with `_initial_set("docks/property_editor/auto_refresh_interval", 0.3);` in `editor_settings.cpp`. But I don't follow what you mean here..

> That means that a restart

What means it needs a restart?

In my testing, I changed the refresh interval value in the editor settings window to 5 seconds and it was immediately applied, and the value in my test scene (see linked issue) was only refreshing every 5 seconds. I then changed it back to 0.3 and the value started refreshing every 0.3 seconds, as expected.

Reply:
No, it's just set with `_initial_set("docks/property_editor/auto_refresh_interval", 0.3);` in `editor_settings.cpp`. But I don't follow what you mean here..

> That means that a restart

What means it needs a restart?

In my testing, I changed the refresh interval value in the editor settings window to 5 seconds and it was immediately applied, and the value in my test scene (see linked issue) was only refreshing every 5 seconds. I then changed it back to 0.3 and the value started refreshing every 0.3 seconds, as expected.

Reply:
Well this is set in `NOTIFICATION_READY`, so it would run only once normally. Or is the EditorInspector recreated when settings change?

Reply:
It gets set both in `NOTIFICATION_READY` and `NOTIFICATION_EDITOR_SETTINGS_CHANGED`

Reply:
Ah indeed, I missed it while reviewing on phone. All good then.